src/convertToObject.js

```diff
@@ -1,18 +1,26 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  
+  const cssRules = sourceString.split(';');
+
+  for (const rule of cssRules) {
+
+    const [property, value] = rule.split(':');
+
+    if (property && value) {
+      
```

Comment:
The if condition is not necessary. If the property or value is undefined, it would not affect the final object.